dashboard/views.py

```diff
@@ -66,9 +66,63 @@ def get(self, request):
 
 class WarningsInstance_View(View):
 
-    def get(self, request):
+    def get(self, request, **kwargs):
+
+        cap_id = kwargs['cap_id']
+        cap_instance = CAPInstance.objects.get(cap_id=cap_id)
```

Comment:
suggestion
```
        try:
            cap_instance = CAPInstance.objects.get(cap_id=cap_id)
        except CAPInstance.DoesNotExist:
            # redirect to not found page
```

Reply:
Thanks Prasana, I will implement these exception handlings

Reply:
Also I can see that the `cap_instance` is only being used for different `.filter` queries. Is it possible to disregard the `cap_instance` part altogether and simply use the `cap_id` in an implementation as mentioned by [this comment](https://github.com/NASA-IMPACT/CDI-Dashboard/pull/7/files#r750706562)

Reply:
This is definitely doable; however in this instance we use other attributes from the CAP instance record in the view and makes sense with the exception handling you suggested